velox/exec/SetAccumulator.h

```diff
@@ -86,6 +86,45 @@ struct SetAccumulator {
     }
   }
 
+  void addNullIndex(const char* buffer) {
+    VELOX_CHECK(!nullIndex.has_value());
+    vector_size_t serializedNullIndex;
+    memcpy(&serializedNullIndex, buffer, kVectorSizeT);
+    if (serializedNullIndex != kNoNullIndex) {
+      nullIndex = serializedNullIndex;
+    }
+  }
+
+  void addSerialized(
+      const FlatVector<StringView>& vector,
+      vector_size_t i,
```

Comment:
nit: i -> index

Perhaps, this method could take StringView or char* or ByteInputStream directly.